### PR TITLE
users: port shell selection to FormSelect

### PR DESCRIPTION
**Pull request by KKoukiou**
The used Select component implementation is deprecated in PF5.

Port this to FormSelect as it's better fit in general (simple Selects in dialogs should use FormSelect) and make it ready for PF5.

### CODE REVIEW

File: pkg/users/account-create-dialog.js [hunk 1]
```diff
@@ -92,16 +92,13 @@ function AccountCreateBody({ state, errors, change, shells }) {
...
-                        selections={shell}
-                        isOpen={isShellSelectExpanded}
-                        aria-labelledby="vm-state-select"
-                        menuAppendTo="parent">
-                    { shells.map(shell_path => <SelectOption value={shell_path} key={shell_path} label={shell_path}>{shell_path}</SelectOption>) }
-                </Select>
+                <FormSelect
+                        data-selected={shell}
+                        id="accounts-create-user-shell"
+                        onChange={selection => { change("shell", selection) }}
```
Comment (cockpituous) [inline pkg/users/account-create-dialog.js:98]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18671-20230419-094958-5a71cc2f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: test/verify/check-users [hunk 2]
```diff
@@ -69,7 +69,7 @@ def createUser(
     run_assert_pixels=False,
 ):
     if default_shell is None:
-        default_shell = getUserAddDetails(machine)["SHELL"]
+        default_shell = getUserAddDetails(machine)["SHELL"] or "/bin/bash"
```
Comment (skobyda) [inline test/verify/check-users:72]: Was this neccesseray because of some test failure?

Comment (KKoukiou) [reply test/verify/check-users:72]: Yes, it was appearing in the tests empty for some distros.

Comment (martinpitt) [reply test/verify/check-users:72]: Yes, Debian-based distros don't user `useradd`, but `adduser`, which has its own config. The default shell there is /bin/sh which is dash, not bash. I haven't checked whether this makes sense (presumably the code has that fallback somewhere as well?).

**Comment by skobyda** [comment]
packit's fedora-38-aarch65 failure at testLvm seems unrelated, so this passes all related tests

**Pull request merged**